### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

File: nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py [hunk 1]
```diff
@@ -206,18 +208,8 @@ def setup_experiment(self, config):
...
-        init_hooks = config.get("init_hooks", None)
-        self.model = create_model(
-            model_class=model_class,
-            model_args=model_args,
-            init_batch_norm=init_batch_norm,
-            device=self.device,
-            init_hooks=init_hooks,
-            checkpoint_file=config.get("checkpoint_file", None)
-        )
+        config["checkpoint_file"] = config.get("checkpoint_file", None)
```
Comment (lscheinkman) [inline nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py:211]: Why is this line necessary?

Comment (subutai) [reply nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py:211]: 🤷
